File: .vscode/striverSDE/Python/largestelement.py
from typing import List

"""
arr: List[int] indicates that the function expects a list of integers as input.
-> int specifies that the function will return an integer.
"""

def sortArr(arr: List[int])-> int:
   arr.sort()
   return arr[-1]



if __name__=="__main__":
   arr1=[1,2,33,23,21]
   arr2 = [8, 10, 5, 7, 9]
   print("The Largest element in the array is:", sortArr(arr1))
   print("The Largest element in the array is:", sortArr(arr2))

File: .vscode/striverSDE/CPP/arrays/largest_element.cpp

#include<iostream>
#include<vector>
#include<bits/stdc++.h>
using namespace std;


// int sortarray(vector<int> &arr){
//     sort(arr.begin(),arr.end());
//     return arr[arr.size()-1];
// }

int sortarray(vector<int> &arr)
{
    int max_element=arr[0];
    for(int i=0;i<arr.size();i++)
    {
        if(arr[i]>max_element)
        {
            max_element=arr[i];
        }
    }
    return max_element;
}

int main(){
    vector<int> arr1={1,2,33,23,21};
    vector<int> arr2={11,12,313,231,121};
    cout<<"the largest element of arr1 "<<sortarray(arr1)<<endl;
    cout<<"the largest element of arr2 "<<sortarray(arr2)<<endl;
    
    return 0;
}